make.js

```diff
@@ -31,6 +32,8 @@ exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
 };
 
 function makeApiFiles(api, sourceDir, apiOutputDir) {
+    var remStaticDefine = ""; // " && !defined(DISABLE_PLAYFAB_STATIC_API)";
```

Comment:
nit: this looks a bit out of place

Reply:
This one is deliberate.
If we set remStaticDefine to the commented value, then the define will be applied to the test projects and libraries, and will immediately take effect across the entire solution.
It's basically a single-toggle that enables or disables this feature.
For customers, it's published "enabled" as is the current default.

---
In reply to: [360617787](https://github.com/PlayFab/XPlatCoreTemplate/pull/69#discussion_r360617787) [](ancestors = 360617787)